jupyter_notebooks/Analyzing public and hybrid cloud networks .ipynb

```diff
@@ -0,0 +1,567 @@
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# traceroute between instances in the same region, using SSH\n",
+    "ans = bfq.traceroute(startLocation=hosts[\"east2_private\"], \n",
+    "                     headers=HeaderConstraints(dstIps=hosts[\"east2_public\"], \n",
+    "                                               applications=\"ssh\")).answer()\n",
+    "show_first_trace(ans.frame())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The trace above shows how traffic goes from `host_east2_private` to `host_east2_public` -- via the source subnet and VPC, then to the transit gateway, and finally to the destination VPC and subnet. Along the way, it also shows where the flow encounters security groups (at both instances) and network ACLs (at subnets). In this instance, all security groups and network ACLs permit this particular flow.\n",
```

Comment:
should replace `host_east2_private` with **host** `east2_private` or something to that effect. having `host_east2_private` in code-font would imply that it is a variable name.